manifests/integreatly-cloud-resources/0.19.0/cloud-resources.v0.19.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,293 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "BlobStorage",
+          "metadata": {
+            "labels": {
+              "productName": "ProductName"
+            },
+            "name": "example-blobstorage"
+          },
+          "spec": {
+            "secretRef": {
+              "name": "example-blobstorage-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Postgres",
+          "metadata": {
+            "labels": {
+              "productName": "productName"
+            },
+            "name": "example-postgres"
+          },
+          "spec": {
+            "secretRef": {
+              "name": "example-postgres-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "PostgresSnapshot",
+          "metadata": {
+            "name": "example-postgressnapshot"
+          },
+          "spec": {
+            "resourceName": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Redis",
+          "metadata": {
+            "labels": {
+              "productName": "productName"
+            },
+            "name": "example-redis"
+          },
+          "spec": {
+            "secretRef": {
+              "name": "example-redis-sec"
+            },
+            "tier": "development",
+            "type": "REPLACE_ME"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RedisSnapshot",
+          "metadata": {
+            "name": "example-redissnapshot"
+          },
+          "spec": {
+            "resourceName": "REPLACE_ME"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/integreatly/cloud-resource-operator:v0.19.0
```

Comment:
don't see this tag in https://quay.io/repository/integreatly/cloud-resource-operator?tab=tags

Reply:
the release is not cut yet, we are waiting, as we might need to expose more metrics, we want to reduce cutting releases if at all possible